docs/wiki/introduction/sql.md

```diff
@@ -480,6 +480,25 @@ There are also encoding functions available, to process query results.
     </p>
     </details>
 
+#### Network functions
+
+- `in_cidr_range(CIDR_RANGE, IP_ADDRESS)`: return 1 if the IP address is within the CIDR range, otherwise 0.
```

Comment:
My inclination is to name this `in_cidr_block`, but maybe that doesn't matter.

Reply:
I've totally agreed with your inclination. I have amended it accordingly.